workshop/posting-error-example.md

```diff
@@ -0,0 +1,199 @@
+The debugging guide also has tips on first steps to take which may help you get to the root of the problem.
+
+**A short list of things to try:**  
+
+- Try to identify which part of your code appears to be the problem through trying smaller parts of the code and seeing if the error still occurs (called chunking).
+In the chunking example below, each highlighted portion is something you could run by itself to test.
+(Using `Command + Enter` in RStudio will run what you have highlighted.)
+
+<img src = "screenshots/chunking_code.png" width = "500">
+
+- Go back to the beginning of your notebook and re-run your code in order and make sure you haven't missed any vital steps! Order matters!
+
+## Step 2) Draft out your question you'll post.
+
+You may want to open up a text editor to write this out before posting (this is a personal preference thing).
+
+For our example error, here's how each of the "four points" (and the optional one) could be included in your posted question; we will go through the details on each point:
+
+<img src = "screenshots/breakdown.png">
+
+### Where is this error occurring; what notebook/chunk?
+
+- In our workshop, notebook references tend to make sense to provide, however, more generally this should be any context around __where__ you've encountered this error.
+
+- If you are *not* working from any CCDL materials, it is *even more important* to provide context around what the notebook you are creating is supposed to be doing.
+
+- Note that line numbers change as you've added code, so line numbers as a reference aren't as helpful.
+
+*For our example:*  
+<img src = "screenshots/where-is-the-error.png" width = "600">
+
+### What is the goal of this code?  
+
+- Providing context around your end goals for this code will help others tailor their advice toward your goal as well as helping the understand what you are working toward.
+    *Examples of informatoni you may want to include (whenever it is applicable):*
+    - What is your scientific question?
+    - What does your input data look like?
+       - What kind of file *e.g.* `.TSV`, `.fastq`?
+       - What information is included the data?
+       - What are the columns?
+       - What are the rows?
+    - What'd your desired output?
+       - What kind of results are you looking for e.g. a plot, a table, processed/filtered data?
+       - What kind of file *e.g.* `.TSV`, `.fastq`?
+
+- Sometimes you can bypass an error completely by using an alternative strategy altogether, but your helper won't be able to suggest an alternative if they don't know your ultimate goal.
+
+*For our example:*  
+<img src = "screenshots/what-is-the-goal.png" width = "600">
+
+### What is the code that is producing this error?
+
+- You should [use backticks to format your code.](../virtual-setup/slack-procedures.md#adding-code-blocks-to-messages).
```

Comment:
```suggestion
- You should [use backticks (```) to format your code.](../virtual-setup/slack-procedures.md#adding-code-blocks-to-messages).
```

Since we can't assume that readers will click the link.